_posts/2019-07-23-event-store-5.0.2-release.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Event Store 5.0.2"
+author: "Shaan Nobee"
+layout: blog-post
+---
+Event Store 5.0.2 is out! It is available for the following operating systems:
+
+- Windows
+- Ubuntu 18.04
+- Ubuntu 16.04
+- Ubuntu 14.04 (EOL reached)
+- macOS 10.9+
+- CentOS 7 (Commercial version)
+- Amazon Linux AMI (Commercial version)
+
+This release comes with some important bug fixes and we highly recommend our users to upgrade as soon as possible. We're also planning to roll out version 4.1.4 with some of these bug fixes within the next few days.
+
+Please note that Ubuntu 14.04 (Trusty Tahr) has reached its end-of-life and thus we will stop releasing packages for it as from the next version.
+
+## Where can I get the packages?
+
+Downloads are available on our [website](https://eventstore.org/downloads/).
+
+The packages can be installed using the following instructions.
+
+**Ubuntu 14.04/16.04/18.04** (via [packagecloud](https://packagecloud.io/EventStore/EventStore-OSS))
+
+```
+curl -s https://packagecloud.io/install/repositories/EventStore/EventStore-OSS/script.deb.sh | sudo bash
+sudo apt-get install eventstore-oss=5.0.2-1
+```
+
+**Windows** (via [Chocolatey](https://chocolatey.org/packages/eventstore-oss/))
+
+```
+choco install eventstore-oss -version 5.0.2
+```
+
+**Client Packages**  
+[EventStore Client](https://www.nuget.org/packages/EventStore.Client/)  
+```
+dotnet add package EventStore.Client --version 5.0.2
+```
+
+[EventStore Embedded Client](https://www.nuget.org/packages/EventStore.Client.Embedded/)  
+```
+dotnet add package EventStore.Client.Embedded --version 5.0.2
+```
+
+## Upgrade Procedure
+To upgrade a cluster, a usual rolling upgrade can be done:
+- Pick a node (Start with slave nodes first, then choose the master last)
+- Stop the node, upgrade it and start it
+
+## Breaking changes
+## HTTP Authorization
```

Comment:
```suggestion
### HTTP Authorization
```

Reply:
hey @ChrisChinchilla , thanks for noticing this. I've intentionally left the indentation at the second level here, to avoid having a small font size for this section's child nodes.